docs/verify.rst

```diff
@@ -126,5 +118,93 @@ any harm:
    ...     @property
    ...     def y(self):
    ...         raise Exception
-   >>> verifyObject(IFoo, Foo())
+   >>> verify_foo()
    True
+
+
+Testing For Methods
+-------------------
+
+Methods are also validated to exist. We'll start by defining a method
+that takes one argument. If we don't provide it, we get an error.
+
+.. doctest::
+
+   >>> class IFoo(Interface):
+   ...    def simple(arg1): "Takes one positional argument"
+   >>> @implementer(IFoo)
+   ... class Foo(object):
+   ...    pass
+   >>> verify_foo()
+   The object <Foo...> has failed to implement interface <InterfaceClass builtins.IFoo>: The IFoo.simple(arg1) attribute was not provided.
```

Comment:
Should it say 'attribute' or 'method' here?

Reply:
I don't think that adds much value and clutters the message.

- Methods *are* attributes; i.e., `getattr(foo, 'bar')` is the same no matter what `bar` "is"
- More specifically, `Method` is a subclass of `Attribute`.
- It's perfectly fine for an object to implement an interface method by defining it as an instance attribute: `self.interface_method = lambda a, b: …`

But I'm looking at it from an experienced implementer's point of view. Maybe it would have value for end-programmers?